pkg/koordlet/metrics/cpu_suppress.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	BESuppressCPU = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
This metric is unused.

Reply:
It's used in `resmanager.cpu_suppress.go`.

Reply:
OK